src/utils/isPRLabelValid.js

```diff
@@ -0,0 +1,19 @@
+// Ignore list of labels for frogtoberfest, add at end of array for more list
+const FROGTOBERFEST_IGNORE_LABEL_LIST = ['invalid'];
+
+// Makes The Array case insensitive for checking
+const labelIgnoreRegex = new RegExp(FROGTOBERFEST_IGNORE_LABEL_LIST.join('|'), 'i');
+
+/**
+ * Checks whether Pull request is valid against Frogtoberfest label list.
+ *
+ * @param {*} labels
+ */
+export default function isPRLabelValid(labels) {
```

Comment:
This is already done by @vhndaree in #16. However we can take some cues from here to improve his approach.